realm_reference_docs_help.md

```diff
@@ -0,0 +1,36 @@
+# Realm Docs Team SDK Reference Manual (API Docs) Collaboration
+
+# Motivation
+
+- API documentation could use some love/attention
+- Docs folks can add an extra set of eyes to make sure that descriptions are current, language is clear, etc.
+- Wordings, sentence structure, descriptions
+
+# Product Description
+
+## Proposed Solution
+
+- docs writers help with API docs, updating them when they notice gaps or inconsistencies
+- tickets to improve API docs independently
+- docs writers as a highly suggested LGTM on SDK work (anywhere that touches code or logic that impacts API docs)
+    - help with wording suggestions
+    - help by asking questions, suggesting additional information to add
+
+## Goals
```

Comment:
Suggestion: add "adhere to docs writer style guide" which is a more descriptive and achievable goal.